docs/querying/query-context.md

```diff
@@ -46,26 +55,28 @@ The query context is used for various query configuration parameters. The follow
 |parallelMergeInitialYieldRows|`druid.processing.merge.task.initialYieldNumRows`|Number of rows to yield per ForkJoinPool merge task for parallel result merging on the Broker, before forking off a new task to continue merging sequences. See [Broker configuration](../configuration/index.html#broker) for more details.|
 |parallelMergeSmallBatchRows|`druid.processing.merge.task.smallBatchNumRows`|Size of result batches to operate on in ForkJoinPool merge tasks for parallel result merging on the Broker. See [Broker configuration](../configuration/index.html#broker) for more details.|
 
+## Query-type-specific parameters
```

Comment:
Should there be a section for datsource specific parameters? To talk about query contexts for join queries (filer pushdown ,etc.) or is that covered in another section

Reply:
Hmm, I'm not sure. I don't think those parameters are documented right now. Should they be or are they 'secret' internal parameters?

Reply:
/cc @jon-wei